app/controllers/concerns/idv/phone_otp_sendable.rb

```diff
@@ -37,7 +37,7 @@ def capture_analytics_for_telephony_error(telephony_error)
         context: 'idv',
         country: Phonelib.parse(send_phone_confirmation_otp_service.phone).country,
       }
-      analytics.track_event(Analytics::OTP_PHONE_VALIDATION_FAILED, attributes)
+      analytics.otp_phone_validation_failed(**attributes)
```

Comment:
we don't need the ** because we can inline the attributes here:
```suggestion
      analytics.otp_phone_validation_failed(
        error: telephony_error.class.to_s,
        message: telephony_error.message,
        context: 'idv',
        country: Phonelib.parse(send_phone_confirmation_otp_service.phone).country,
      )
```

Reply:
Hi... unresolving this, it looks like the change is not in the diff

Reply:
Ah not there.

Reply:
this is still not inlined?

Reply:
It's inlined now.